docs/examples.md

```diff
@@ -0,0 +1,57 @@
+# Examples generation
+
+Prism will try to generate meaningful examples for all the responses and use some fallback mechanisms in case the generation was not possible. This document outlines how the process works so that you will understand why Prism has responded with such example.
+
+By default, Prism will use a **static generation strategy**, which is outlined below. You can enable the dynamic examples generation by using the `-d` flag in the command line.
+
+`prism mock api.oas3.yaml` -> Static example generation
+
+`prism mock -d api.oas3.yaml` -> Dynamic example generation
+
+## Foreword
+
+We will assume you're already familiar with the mocking flow. In case you're not, you can consult [our diagram][diagram]. This document describes fundamentally what happens once we have enough data to return a meaningful payload (aka: the last green circle).
+
+When the negotiator has sorted out the response to return to the client and has selected the appropiate `content` according to the `Accept` header, but there's no element in the `examples` property, then the example generation kicks in.
+
+Both Dynamic and Static generation almost follow the same path, which forks in the final behaviour in case anything else fails.
+
+## Static Example Generation
+
+1. If the provided JSON Schema has an example for the whole object, it will return such value (or the first one in case it is an array)
+2. If not, for each property
+  * If the property has a default value, then it will return the specified value (or the first one in case it's an array)
```

Comment:
The example keyword should only have an array if the property is an array, in such case the whole array should be used. If it’s not an array, having an array in the example is invalid. 

You might be thinking of the examples keyword?